packages/shared/src/components/cards/v1/RaisedLabel.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import classed from '../../../lib/classed';
+
+export enum RaisedLabelType {
+  Hot = 'Hot',
+  Pinned = 'Pinned',
+  Beta = 'Beta',
+}
+
+const typeToClassName: Record<RaisedLabelType, string> = {
+  [RaisedLabelType.Hot]: 'bg-theme-status-error',
+  [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-cabbage-40',
+};
+
+export interface RaisedLabelProps {
+  type: RaisedLabelType;
+  description?: string;
+  className?: string | undefined;
+}
+
+const Spacer = classed(
+  'div',
+  'mt-[7px] w-2 h-1 bg-theme-bg-primary  group-hover:bg-theme-bg-secondary group-focus:bg-theme-bg-secondary group-focus:mt-1.5',
+);
+
+export function RaisedLabel({
+  type = RaisedLabelType.Hot,
+  description,
+  className,
+}: RaisedLabelProps): ReactElement {
+  return (
+    <div
+      className={classNames(
+        'absolute -top-2 right-2 flex flex-row px-2',
+        className,
+      )}
+    >
+      <Spacer />
```

Comment:
Can you tell us more of the purpose of this spacer?

Reply:
If you look at the designs for hover/focus with the flag shown:
<img width="241" alt="Screenshot 2024-01-08 at 16 27 28" src="https://github.com/dailydotdev/apps/assets/9974711/ef6dc45b-4aac-4a76-9afa-514e8784a26b">

There is a gap between the flag and the top border on the card. The spacer is there to cover the border, so that it looks the same as in the design. I could maybe do the same trick by having a px-2 and playing with a background gradient on the flag div as well, if you'd prefer that instead. This just was faster to come up with on my end.

Reply:
Could be done with a wrapper div as well rather then with 2 different divs - `Spacer` set in the beginning and end

Reply:
Yup. I'll probably play with the background gradient. That way we can avoid having additional DOM elements altogether (hopefully).

Reply:
I did away with the spacers and instead implemented it with wrapper div being absolutely positioned to `top: -1px` (or `top: -2px` on focus) and relatively positioning the child div with the flag to `top: -8px`.